src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -199,7 +200,8 @@ def default_api_version(self):
             'classic_administrators': '2015-06-01'
         }),
         ResourceType.DATA_KEYVAULT: '2016-10-01',
-        ResourceType.DATA_STORAGE: '2015-04-05'
+        ResourceType.DATA_STORAGE: '2015-04-05',
+        ResourceType.MGMT_MANAGEDSERVICES: '2018-06-01-preview'
```

Comment:
You only need to add MGMT_MANAGEDSERVICES here only if 2017-03-09-profile needs to support managed service commands.

Reply:
I see. I mistook it for the api version.